packages/turf-shortest-path/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@turf/shortest-path",
+  "version": "4.0.0",
+  "description": "turf shortest-path module",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
+  "scripts": {
+    "test": "node test.js",
+    "bench": "node bench.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Turfjs/turf.git"
+  },
+  "keywords": [
+    "turf",
+    "shortest-path",
+    "path"
+  ],
+  "author": "Turf Authors",
+  "contributors": [
+    "Stefano Borghi <@stebogit>"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Turfjs/turf/issues"
+  },
+  "homepage": "https://github.com/Turfjs/turf",
+  "devDependencies": {
+    "benchmark": "^2.1.4",
+    "write-json-file": "^2.2.0",
+    "load-json-file": "^2.0.0",
+    "tape": "^4.6.3"
+  },
+  "dependencies": {
+    "@turf/turf": "4.7.3",
+    "easystarjs": "0.4.1",
+    "grid-to-matrix": "1.3.0",
+    "javascript-astar": "0.4.1",
+    "pathfinding": "0.4.18"
+  }
```

Comment:
There's definitely missing some dependencies in the `package.json`, this is one of the reasons why the TravisCI tests will fail.

Reply:
All this is absolutely **"work-in-progress"**, I added the whole `@turf/turf` for convenience while developing. 
Sorry, I should have titled this more clearly as "proposal" (like the label); I was just interested at this point in you suggestions and check with you if my approach was reasonably sound so far.

Reply:
👍 Looks great to me so far! Was just trying to get the tests to pass